app/models/goal.py

```diff
@@ -1,6 +1,21 @@
 from flask import current_app
 from app import db
+from sqlalchemy.orm import relationship
```

Comment:
hmm.. I don't think you need this import from sqlalchemy as you're already importing and initializing it as `db` in `__init__.py`. 
This means `relationship` is coming from `db` rather than `sqlalhemy.orm` in this file.